lib/VersionHistory/getFieldLabels.js

```diff
@@ -1,17 +1,23 @@
 import { escapeRegExp } from 'lodash';
 
+import { HIDDEN_FIELDS_MAP } from './constants';
+
 export const getFieldLabels = (intl, paths, labelsMap = {}) => {
   if (!paths) return null;
 
   const labelsMapEntries = Object.entries(labelsMap);
 
-  return paths.map((path) => {
+  return paths.reduce((acc, path) => {
     const fieldLabel = labelsMapEntries.find(([fieldPath]) => {
       const regex = new RegExp(`^${escapeRegExp(fieldPath).replace('\\\\d', '\\d')}$`);
 
       return regex.test(path);
     })?.[1] || path;
 
-    return intl.formatMessage({ id: fieldLabel });
-  }).sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()));
+    if (!HIDDEN_FIELDS_MAP[fieldLabel]) {
```

Comment:
A list of the fields to hide should come as an argument to the function and shouldn't be binned with the PO Line entity.